File: 14_02_2024_Rearrange_Array_Elements_by_Sign.cpp
class Solution {
public:
    vector<int> rearrangeArray(vector<int>& nums) {
        int l=nums.size();
        vector<int> v(l);
        int k=0;
        for(int i=0;i<l;i++){
            if(nums[i]>0){
                v[k]=nums[i];
                k+=2;
            }
        }
        k=1;
        for(int i=0;i<l;i++){
            if(nums[i]<0){
                v[k]=nums[i];
                k+=2;
            }
        }
        return v;
       
    }
};

File: 14_02_2024_Rearrange_Array_Elements_by_Sign.py
class Solution:
    def rearrangeArray(self, nums):
        l = len(nums)
        v = [0] * l
        k = 0
        for i in range(l):
            if nums[i] > 0:
                v[k] = nums[i]
                k += 2
        k = 1
        for i in range(l):
            if nums[i] < 0:
                v[k] = nums[i]
                k += 2
        return v

